action.yml

```diff
@@ -174,23 +174,9 @@ outputs:
 runs:
   using: 'composite'
   steps:
-    - name: "Extract Docker image version"
-      id: extract-version
-      shell: bash
-      run: |
-        # Extract version from github.action_ref
-        # Example: "web-id-fr/forge-review-app-action@v1.3.2" -> "v1.3.2"
-        if [[ "${{ github.action_ref }}" == *"@"* ]]; then
-          VERSION="${{{ github.action_ref }}#*@}"
-        else
-          VERSION="latest"
-        fi
-        echo "version=$VERSION" >> $GITHUB_OUTPUT
-        echo "Using Docker image version: $VERSION"
-
     - name: "Forge Review-app Action"
       id: forge-review-app-action
-      uses: "docker://ghcr.io/web-id-fr/forge-review-app-action:${{ steps.extract-version.outputs.version }}"
+      uses: "docker://ghcr.io/web-id-fr/forge-review-app-action:v1"
```

Comment:
couldn't we just `bash entrypoint.sh`? 🤔 

when it's like this I can't override and test a directly from git version in a branch 😞

Reply:
Indeed, it was a bad idea to use "pre-built" docker image, cause we do not have control on the version used in this `action.yml`.

So, I have reworked a lot of things, added the bats-core testing framework with automated testing on branch push. Details here : https://github.com/web-id-fr/forge-review-app-action/pull/28

Now, if you do `uses: web-id-fr/forge-review-app-action@main` in your github workflow, it will build the docker image on the fly before using it, with the proper branch source code loaded in 🙂

Reply:
amazing, ty!!